interactive_engine/compiler/conf/ir.compiler.properties

```diff
@@ -16,7 +16,7 @@ pegasus.hosts: localhost:1234
 # graph.schema
 graph.schema: ../executor/ir/core/resource/modern_schema.json
 graph.store: exp
-graph.planner: {"isOn":true,"opt":"RBO","rules":["FilterMatchRule"]}
+graph.planner: {"isOn":true,"opt":"RBO","rules":["FilterIntoJoinRule", "FilterMatchRule", "NotExistToAntiJoinRule"]}
```

Comment:
`NotExist` looks like a gremlin semantics. How about `NotMatch`?

Reply:
fix in https://github.com/alibaba/GraphScope/pull/3088/commits/03fed995ff05e005d1b37ca44ad19046dfa5ee24